.github/actions/upgrade-indexer/upgrade-indexer.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Gets the absolute path of the script, used to load the common.sh file
+function get_absolute_path() {
+    RELATIVE_PATH="$(dirname -- "${BASH_SOURCE[0]}")"
+    ABSOLUTE_PATH="$(cd -- "$RELATIVE_PATH" && pwd)"
```

Comment:
Is it really necessary to use the `cd` command? I seem to remember that this type of command was changed in version 4.2 of the Wazuh installation assistant to avoid certain problems.

Reply:
True. Done. Tested in: https://github.com/davidcr01/wazuh-packages/actions/runs/3845237759/jobs/6549151074